src/modules/github/components/bulk-pull-reviews/bulk-pull-reviews.tsx

```diff
@@ -39,6 +43,15 @@ const BulkPullReviewsLayout = ({
 const Repositories = () => {
   const { isLoading } = useFetchRepositories()
   const { repos } = useRepos()
+  const { translate } = useDictionary()
+  const { openAllRepo } = useUpdateRepoOrPull()
+
+  const handleExpandAllClick = () => {
+    if (!repos) {
+      return
```

Comment:
we do not need this condition here. if the `repos` variable is not defined, it will be covered in the reducer.